### PR TITLE
DOC: Update Kurt Docstr

### PR DESCRIPTION
**Pull request by WillAyd**
Continuation of #19999

### CODE REVIEW

**Comment by WillAyd** [comment]
@jorisvandenbossche

File: pandas/core/window.py [hunk 1]
```diff
@@ -910,8 +910,9 @@ def skew(self, **kwargs):
 
     Returns
     -------
-    Series or DataFrame (matches input)
-        Like-indexed object containing the result of function application
+    Series or DataFrame
+        Returned object type is dictated by the caller of the %(name)s
+            calculation
```
Comment (jorisvandenbossche) [inline pandas/core/window.py]: align with the "Returned .." from above

File: pandas/core/window.py [hunk 2]
```diff
@@ -910,8 +910,9 @@ def skew(self, **kwargs):
 
     Returns
     -------
-    Series or DataFrame (matches input)
-        Like-indexed object containing the result of function application
+    Series or DataFrame
+        Returned object type is dictated by the caller of the %(name)s
```
Comment (jorisvandenbossche) [inline pandas/core/window.py]: As a non-native speaker, I found your original "determined" better than "dictated" (but maybe not more correct :-)). Or simpler "Returned object is of the same type as the caller ..." ?

File: pandas/core/window.py [hunk 3]
```diff
@@ -932,19 +933,20 @@ def skew(self, **kwargs):
...
     3.999946
-    >>> df = pd.DataFrame(arr)
-    >>> df.rolling(4).kurt()
-              0
-    0       NaN
-    1       NaN
-    2       NaN
-    3 -1.200000
-    4  3.999946
+    >>> ser = pd.Series(arr)
```
Comment (jorisvandenbossche) [inline pandas/core/window.py]: let's use `s` by default (following the guide)

**Review by jorisvandenbossche — COMMENTED** [review]
Looks good! some minor comments

**Comment by WillAyd** [comment]
Travis failures is from `TestTSPlot` and isn't related to this change

**Comment by toobaz** [comment]
Looks good!

**Comment by jorisvandenbossche** [comment]
@toobaz do we merge this one first, and then you update your PR?

**Comment by toobaz** [comment]
Yes, I will rebase, no problem.

**Comment by jorisvandenbossche** [comment]
Just made a small change to "Like" -> "Equivalent" (I *think* you cannot use "like" as an adjective like that? but may be wrong)

**Comment by jorisvandenbossche** [comment]
@WillAyd Thanks for the updates!

**Pull request merged**